### PR TITLE
[charts/csi-isilon]Added Powerscale 2.7.0 Support

### PR DESCRIPTION
**Pull request by chimanjain**
#### Is this a new chart?

No

#### What this PR does / why we need it:

It is to support CSI-Powerscale 2.7.0 driver

#### Which issue(s) is this PR associated with:

- #Issue_Number

#### Checklist:

- [x] Chart Version bumped
- [ ] Variables are documented in the chart README.md
- [x] Title of the PR starts with the chart name (e.g. `[charts_dir/mychartname]`) if applicable

### CODE REVIEW

File: charts/csi-isilon/values.yaml [hunk 1]
```diff
@@ -365,37 +368,51 @@ authorization:
   # skipCertificateValidation: certificate validation of the csm-authorization server
   # Allowed Values:
   #   "true" - TLS certificate verification will be skipped
-  #   "false" - TLS certificate will be verified 
-  # Default value: "true" 
+  #   "false" - TLS certificate will be verified
+  # Default value: "true"
   skipCertificateValidation: true
 
+# Storage Capacity Tracking
```
Comment (niranjan-n1) [inline charts/csi-isilon/values.yaml:375]: storageCapacity: enabled :true should be added in charts/container-storage-modules/values.yaml as well

**Review by niranjan-n1 — CHANGES_REQUESTED** [review]
1 comment

File: charts/csi-isilon/values.yaml (hunk 1 above)
Comment (niranjan-n1) [reply charts/csi-isilon/values.yaml:375]: also kindly update the same in this repository as well https://github.com/dell/csm-docs/blob/release-1.7.0/content/docs/deployment/csminstallationwizard/src/templates/helm/csm-1.7.0-values.template

Comment (chimanjain) [reply charts/csi-isilon/values.yaml:375]: Ok

**Review by nitesh3108 — APPROVED** [review]

**Review by niranjan-n1 — COMMENTED** [review]
LGTM

**Review by niranjan-n1 — APPROVED** [review]
LGTM

**Pull request merged**